spec/support/system_helpers.rb

```diff
@@ -180,6 +180,9 @@ def fake_scroll_down_long(selector_to_make_tall = "#main-outlet")
     page.execute_script(
       "document.querySelector('#{selector_to_make_tall}').style.height = '10000px'",
     )
+
+    sleep 0.1 # give time to browser
```

Comment:
This seems like a potential flake. How about something like:
```suggestion
    try_until_success do 
      expect(page.evaluate_script("window.scrollY")).to eq(1000)
    end
```

(similar to https://github.com/discourse/discourse/blob/a91f8cee0a426992ec6824dbbdc7d137102d1a79/spec/system/scroll_manager_service_spec.rb#L31C39-L31C39)

Reply:
yes i had tried similar stuff, but was still flakey, will try this

Reply:
I went back to the sleep, your suggested fix was failing,